site/docs/v1/tech/guides/breached-password-detection.adoc

```diff
@@ -0,0 +1,274 @@
+* Flip a flag on their account to more closely monitor it for suspicious activity in the future
+
+All of these require custom code on your end, of course, but the webhook can trigger any custom action.
+
+==== Sending an Email
+
+Sending the user an email lets the user know that their account could be compromised, but doesn't force them to take any action.
+
+This email is an link:../email-templates/[email template] which can be customized and localized similar to any other template. You'll typically include a link to reset their password, but you may want to include other information as well. Below is the default HTML template.
+
+.HTML Breached Password Email Template
+[source,text]
+----
+include::docs/src/email/breached-password.html[]
+----
+
+Please see the link:/docs/v1/tech/email-templates/email-templates#breached-password[email template documentation] for more information, including available template variables.
+
+==== Requiring a Password Change
+
+The final option for handling breached passwords at user sign in is to require a password change. Once a password has been checked and found to be breached, the account is marked in the system as requiring a changed password. The user will be forced to change their password whenever they attempt to login. Be aware that changing the 'On login' setting to a different option will have no effect on accounts so marked.
+
+When this option is enabled, if you are using the FusionAuth templates, the page displayed to the user after they login with a compromised password is is generated by the "OAuth Change password form" template. As with any other template, you can customize it, including look and feel and any messages displayed. Please check out the link:../email-templates/[themes documentation] for more information.
+
+If you are instead using the link:../apis/login[login API], the API call will respond with a `203` HTTP status code. Please review the API documentation for more information. 
+
+You can also read the <<End User Experience>> section for a deeper understanding of what the end user will encounter.
+
+=== Recommended Settings
+
+The recommended options for maximum protection from compromised passwords are:
+
+* a *Match mode* of *High*
+* setting the *On login* option to require a password change. Alternatively, set this value to only record the result and configure a webhook to process password breach events in a custom manner.
+
+Of course, choose options that work for you and your system. This is why FusionAuth provides the configuration choices outlined above. 
+
+== Breached Password Detection Reporting
+
+Reporting is a key part of breached password detection. Aggregate information across all your users can help inform which actions to take on detection, both within FusionAuth and in external systems. 
+
+Reports also let you know which users have a breached account credentials. This allows you contact or take other actions on the compromised account.
```

Comment:
... allows you to **take** contact or ~take~ other actions...

Reply:
Hmm. I'm not sure you "take contact". I'll rework a bit to be clearer.

Reply:
Ha.. yes, you're right. That was a bad suggestion. But perhaps there is a way to rework it to read better. Thanks!